backend/src/controllers/userControllers.js

```diff
@@ -65,7 +65,8 @@ const add = (req, res, next) => {
     })
     .catch((err) => {
       console.error(err);
-      res.sendStatus(500);
+
+      res.status(500).send(err.sqlMessage);
```

Comment:
don't send sql messages to the front: hackers don't need clues about your database ;)